libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -52,12 +52,12 @@ public static NewSlackMessage ActivityToSlack(Activity activity)
             }
 
             // should this message be sent as an ephemeral message
-            if (message.Ephemeral != null)
+            if (!string.IsNullOrEmpty(message.Ephemeral))
```

Comment:
Can we use `string.IsNullOrWhiteSpaces` instead?

Reply:
I think it's a good idea